app/services/autocomplete_search_service.rb

```diff
@@ -26,7 +26,25 @@ def find_suggestions # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
       request.params[:q] = CGI.escape(@search_term)
     end
 
-    results = response.body.split("\n")
+    # The response from the autocomplete service sets the HTTP header:
+    #
+    #   Content-Type: text/html; charset=ISO-8859-1
+    #
+    # but in reality the body string is UTF-8.
+    #
+    # For example, the results for search term "wha" include strings with
+    # macrons e.g. "Whangārei". By inspection of the content in a hex editor
```

Comment:
I think the comment is less clear without the macron. Ruby's default encoding since 2.0 has been UTF-8 so AFAIK it should be fine with non ASCII characters in comments. That said, I'm not  welded to this opinion so will remove it if others think that is better.

Reply:
I think the comment is less clear without the macron. Ruby's default encoding since 2.0 has been UTF-8 so AFAIK it should be fine with non ASCII characters in comments. That said, I'm not welded to this opinion so will remove it if others think that is better.

Reply:
I think the comment is less clear without the macron. Ruby's default encoding since 2.0 has been UTF-8 so AFAIK it should be fine with non ASCII characters in comments. That said, I'm not welded to this opinion so will remove it if others think that is better.

Reply:
I think this needs to change in Rubocop. They're really trying to stop people using the long hyphens and smart quotes when they mean to use the more common ones that were in ASCII. It's a bit anglocentric IMO.

Reply:
I think it adds context so leave the comment in.